projects/canopy/src/lib/carousel/carousel-item/carousel-item.component.ts

```diff
@@ -18,6 +21,15 @@ export class LgCarouselItemComponent {
   @HostBinding('attr.role') role = 'tabpanel';
   @HostBinding('attr.aria-selected') public selected = false;
 
+  @Output() pauseEmit = new EventEmitter<string>();
+
+  @HostListener('click', ['$event.target.type'])
```

Comment:
Will this trigger when a button is 'clicked' with a keyboard?

Reply:
Good point, `click` usually covers the `enter` key but I don't think it covers the space 🤔